src/components/Product/Product.js

```diff
@@ -1,29 +1,29 @@
 import styles from './Product.module.scss';
-//import clsx from 'clsx';
 import PropTypes from 'prop-types';
-import { useState } from 'react';
+import { useState, useMemo } from 'react';
 import ProductImage from '../ProductImage/ProductImage';
-//import shortid from 'shortid';
 import ProductForm from '../ProductForm/ProductForm'
 
 const Product = ({title, basePrice, colors, sizes, name }) => {
 
 const [currentColor, setCurrentColor] = useState(colors[0]);
 const [currentSize, setCurrentSize] = useState(sizes[0].name);
 
-const getPrice = () => {
+const getPrice = useMemo(() => {

```

Comment:
useMemo zwraca w tej chwili zapamiętaną wartość. Wcześniej to była funkcja (stąd `getPrice`). Teraz jest to po prostu wartość, więc nazwa powinna wskazywać na zmienną przechowującą wartość (getPrice sugeruje, że to funkcja). Zmień nazwę na `price`

Reply:
Nazwa zmieniona z `getPrice` na `price` .